File: Linked List/CircularLL.cpp
#include <iostream>
using namespace std;

class Node
{
public:
    int data;
    Node *next;
};

class CircularLL{
    private:
    Node *head;
    public:
    CircularLL(){
        head = NULL;
    }
    void Create(){
        int n, el;
        cout << "Enter no. of elements: ";
        cin >> n;
        for (int i = 0; i<n; i++){
            cout << "Enter element " << i+1 << ": ";
            cin >> el;
            Node *newNode;
            newNode = new Node();
            newNode->data = el;
            if (head==NULL){
                head = newNode;
                newNode->next = newNode;
            }
        }
    }

    void Display(){
        Node *ptr;
        ptr = head;
        do {
            cout << ptr->data << ' ';
            ptr = ptr->next;
        } while (ptr!=head);
    }
};

int main(){
    CircularLL ll;
    ll.Create();
    ll.Display();
    return 0;
}

File: Linked List/CircularLL.py
class Node:
    def __init__(self,data):
        self.data = data
        self.next = None
        
class CircularLL:
    def __init__(self):
        self.head = None
        
    def Create(self):
        n = int(input("Enter no. of elements: "))
        for i in range(n):
            el = int(input(f"Enter element {i+1}: "))
            newNode = Node(el)
            if self.head==None:
                self.head = newNode
                newNode.next = newNode
                last = newNode
            else:
                last.next = newNode
                newNode.next = self.head
                last = newNode
    
    def Display(self):
        ptr = self.head
        while(ptr):
            print(ptr.data, end=" ")
            ptr = ptr.next
            if ptr==self.head: break
            
cll = CircularLL()
cll.Create()
cll.Display()